src/main/java/org/spongepowered/api/item/inventory/Inventory.java

```diff
@@ -145,9 +145,7 @@
      * Adds an ItemStack to this inventory.
      * Returns a {@link InventoryTransactionResult.Type#SUCCESS} only if the entire {@link ItemStack} fits the inventory.
      *
-     * <p>The size of the supplied stack is reduced by the number of items successfully consumed by the inventory.</p>
```

Comment:
Why remove this? IMO it was very convenient to use it like this. E.g. I can just a itemstack to a list of inventories and it will be added until it's consumed.

For the `set` method it makes sense to remove it.

Reply:
You could also do this by offering to the union of your list of inventories.

Reply:
And to answer why because @Zidane 
fight!